libraries/AP_Relay/AP_Relay.cpp

```diff
@@ -140,6 +141,13 @@ void AP_Relay::set(const uint8_t instance, const bool value)
     if (_pin[instance] == -1) {
         return;
     }
+    _pin_states = value?_pin_states | (1<< instance): _pin_states & ~(1<< instance);
+    AP::logger().Write("RELY", "TimeUS,State", "QB",
+                                        AP_HAL::micros64(),
+                                        _pin_states);
+#if AP_SIM_ENABLED
+    return;
```

Comment:
why don't we want them to actuate on real hardware?

Reply:
safety issue.....sim-on-hardware is supposed to default to no actuator operation unless intentionally recompiled to remove the motor/serv, and now, relay blocks

Reply:
looks like this has broken SITL. I think you would also need to check board type is not SITL. 

`#if AP_SIM_ENABLED && (CONFIG_HAL_BOARD != HAL_BOARD_SITL) `

Reply:
will do